incubator/airflow/values.yaml

```diff
@@ -0,0 +1,332 @@
+    ##
+    ##              location /flower/ {
+    ##                rewrite ^/flower/(.*)$ /$1 break;
+    ##                proxy_pass http://example.com:5555;
+    ##                proxy_set_header Host $host;
+    ##              }
+    ##            }
+    ## ==== IF YOU HAVE FLOWER > JANUARY 2018, NO MORE NEED TO STRIP THE PREFIX ====================
+    annotations:
+      ##
+      ## NOTE: it is important here to have your reverse proxy strip the path/rewrite the URL
+      ## Example for Traefik:
+      # traefik.frontend.rule.type: PathPrefix       ## Flower >= Jan 2018
+      # traefik.frontend.rule.type: PathPrefixStrip  ## Flower < Jan 2018
+      # kubernetes.io/ingress.class: traefik
+    tls:
+      ## Set to "true" to enable TLS termination at the ingress
+      enabled: false
+      ## If enabled, set "secretName" to the secret containing the TLS private key and certificate
+      ## Example:
+      ## secretName: example-com-crt
+
+
+##
+## Storage configuration for DAGs
+persistence:
+  ##
+  ## enable persistance storage
+  enabled: false
+  ##
+  ## Existing claim to use
+  existingClaim: nil
+  ##
+  ## Persistent Volume Storage Class
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+  ##   GKE, AWS & OpenStack)
+  # storageClass: default
+  accessMode: ReadWriteOnce
+  ##
+  ## Persistant storage size request
+  size: 1Gi
+
+
+##
+## Configure DAGs deployment and update
+dags:
+  ##
+  ## mount path for persistent volume.
+  ## Note that this location is referred to in airflow.cfg, so if you change it, you must update airflow.cfg accordingly.
+  path: /usr/local/airflow/dags
+  ##
+  ## Set to True to prevent pickling DAGs from scheduler to workers
+  donot_pickle: false
+  ##
+  ## Configure Git repository to fetch DAGs
+  git:
+    ##
+    ## url to clone the git repository
+    url:
+    ##
+    ## branch name, tag or sha1 to reset to
+    ref: master
+  init_container:
+    ## Fetch the source code when the pods starts
+    enabled: false
+    ## install requirements.txt dependencies automatically
+    install_requirements: true
+
+##
+## Configuration values for the postgresql dependency.
+## ref: https://github.com/kubernetes/charts/blob/master/stable/postgresql/README.md
+postgresql:
+  ##
+  ## Use the PostgreSQL chart dependency.
+  ## Set to false if bringing your own PostgreSQL.
+  enabled: true
+  ##
+  ## If bringing your own PostgreSQL, the full uri to use
+  ## e.g. postgres://airflow:changeme@my-postgres.com:5432/airflow?sslmode=disable
+  # uri:
+  ##
+  ### PostgreSQL User to create.
+  postgresUser: postgres
+  ##
+  ## PostgreSQL Password for the new user.
+  ## If not set, a random 10 characters password will be used.
+  postgresPassword: airflow
```

Comment:
Don't put passwords on the `values.yaml` file. Create a directory `ci`, and a `example-values.yaml` file with these values so that your chart can install in case it needs them. https://github.com/helm/charts/blob/master/test/README.md#providing-custom-test-values